src/frontend/src/utils/findWebAuthnRpId.ts

```diff
@@ -0,0 +1,107 @@
+import { DeviceData } from "$generated/internet_identity_types";
+
+const defaultDomain = "https://identity.ic0.app";
```

Comment:
Should probably be all caps since it's a variable constant?

```suggestion
const DEFAULT_DOMAIN = "https://identity.ic0.app";
```